Assets/MRTK/SDK/Editor/Inspectors/UX/InteractiveElement/InteractiveElementInspector.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License
+
+using Microsoft.MixedReality.Toolkit.UI.Interaction;
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.Editor
+{
+    /// <summary>
+    /// Custom inspector for an InteractiveElement.
+    /// </summary>
+    [CustomEditor(typeof(InteractiveElement))]
+    public class InteractiveElementInspector : BaseInteractiveElementInspector
+    {
+        protected override void OnEnable()
+        {
+            base.OnEnable();
+        }
+
+        public override void OnInspectorGUI()
+        {
+            // Interactive Element is a place holder class
```

Comment:
I am wondering if the comment should be moved up to around line 10. Also not sure if we need these two overrides given they just call the base.

Reply:
Yes, the override methods are not necessary in this case, thanks for the catch!